### PR TITLE
Import TA2 S4 model

### PR DESCRIPTION
**Pull request by Keno**

### CODE REVIEW

File: docs/src/scenarios/scenario4.md [hunk 1]
```diff
@@ -121,7 +231,9 @@ plot(s2, idxs = [IS])
 ```
 
 ```@example scenario4
-p_opt, s2, ret = optimal_parameter_threshold(prob, D, 430, u_test, [u_test], [0.0], [NN],
+p_opt, s2, ret = optimal_parameter_threshold(prob, D, 430, u_test, [u_test], [0.0], [
```
Comment (YingboMa) [inline docs/src/scenarios/scenario4.md:234]: Could you update the names like `u_test` here?

File: docs/src/scenarios/scenario4.md [hunk 2]
```diff
@@ -58,47 +58,157 @@ tdays = (tend - tstart).value
...
-                              :rec => (:I => :R),
-                              :test => (:I => :IS),
-                              :leave => (:IS => :R),
-                              :death => (:I => :D))
-    return SEIRHD
+using AlgebraicPetri
+using ModelingToolkit
+using Catlab
+using Catlab.CategoricalAlgebra, Catlab.Programs.RelationalPrograms, Catlab.WiringDiagrams.WiringDiagramAlgebras
+using ASKEM.Dec2022Demo: typed_stratify
```
Comment (YingboMa) [inline docs/src/scenarios/scenario4.md:65]: How can we install `ASKEM`?

File: docs/src/scenarios/scenario4.md [hunk 3]
```diff
@@ -58,47 +58,157 @@ tdays = (tend - tstart).value
...
+seir_test_iso_cohort = typed_stratify(seir_test_iso_typed, cohort_typed)
+
+sys = ODESystem(map(
+        dom(seir_test_iso_cohort),
+        Name=t -> Symbol(string(t[1]) * "_" * string(t[2]))
+    ))
+
+# TODO: TA1 needs to hand off to TA2
+ps = parameters(sys)
+rates = ps .=> rand(length(ps)) * NN
```
Comment (YingboMa) [inline docs/src/scenarios/scenario4.md:201]: Maybe we need to make it at least deterministic.

File: docs/src/scenarios/scenario4.md (hunk 2 above)
Comment (ChrisRackauckas) [reply docs/src/scenarios/scenario4.md:65]: https://github.com/SciML/EasyModelAnalysis.jl/pull/104

**Pull request closed without merging**